strr-platform-web/app/locales/en-CA.ts

```diff
@@ -72,30 +79,35 @@ export default {
-      listingSize: 'What is the total number of listings offered by the Platform Provider on June 1, {year}?',
+      isUserRep: 'Are you the platform service provider representative? ',
+      listingSize: 'What is the total number of listings offered by the Platform Service Provider across all platforms on June 1, {year}?',
       nonComplianceEmail: 'Provide an email address to receive Notices of Non-Compliance.',
       nonComplianceEmailLong: 'This message will be sent to the supplier host and platform service provider by a local government to inform both parties that a listing is not compliant with a local government business licence requirement. No action is required from the platform service provider.',
-      primaryContact: 'This is the primary contact person for the platform.',
+      primaryContact: 'This is the primary contact person for the platform service provider.',
       takedownEmail: 'Provide an email address to receive Takedown Requests.',
       takedownEmailLong: 'This message will be sent to the platform service provider by a local government to request the platform service provider cease providing platform services for a listing that is not compliant with a local government business licence requirement. The request may be sent within a period of 5-90 days after a Notice of Non-compliance was delivered. As per s.18 (3)(b) of the Short-term Rental Accommodations Act, platform service providers must comply with the request of the local government.',
       thousandOrMore: '1000 or more',
       '250-999': '250-999',
       '249orLess': '249 or less',
       selectAccountForStrr: 'Select the account you wish to use to register your Short-term rental platform.',
       onlyPremiumAccountWarning: '{boldStart}Note:{boldEnd} Only Premium accounts using Pre-authorized Debit (PAD) can be used to register Short-term Rental Registry Platforms.',
-      onlyPremiumAccountModalContent: 'You must create a BC Registries Premium Account that uses a Pre-authorized Debit (PAD) payment method.'
+      onlyPremiumAccountModalContent: 'You must create a BC Registries Premium Account that uses a Pre-authorized Debit (PAD) payment method.',
+      listEachPlatform: 'Please list each platform operated by the platform service provider.'
     },
     review: {
       platInfo: {
-        brandName: 'Platform Brand Name | Platform Brand {count} Name',
-        brandSite: 'Platform Brand Website | Platform Brand {count} Website',
+        platformName: 'Platform Name | Platform {count} Name',
+        platformSite: 'Platform Website | Platform {count} Website',
         size: 'Platform Size',
         sizeDesc: {
           GREATER_THAN_THOUSAND: 'Greater than 1,000 listings',
           LESS_THAN_THOUSAND: 'Less than 1,000 listings'
         }
       }
     }
+  },
+  validation: {
+    platformName: 'Please enter a platform name',
+    platformSite: 'Please enter a valid full URL for this platform (i.e. https://www.bcregistry.gov.bc.ca)'
```

Comment:
if this is set as 'brandName' + 'brandSite' then I think you can remove the 'getPlatformSchema' ?